### DISCUSSION TITLE
Python Web Scraping Challenge: Downloading Images | Trending Tutorial

### DISCUSSION THREAD

**Discussion by ruthrina** [community/community#90014] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Unlock the power of Python with our latest challenge! Dive into the world of web scraping as we guide you through scraping and downloading images using Python, BeautifulSoup, and the requests library.

https://youtu.be/mNSmlqv9q_c

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mzums** [comment]
Thanks for info, looks like a great challenge. I'll be happy to participate

**Comment by queenofcorgis** [comment]
Hi @ruthrina , we’re glad you’re here 😀

Thanks for sharing your video with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
